ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.11.yaml

```diff
@@ -51,6 +51,29 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: check-aws-no-capabilities
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    env:
+      BASELINE_CAPABILITY_SET: None
+    test:
+    - as: test
+      commands: |
+        function assert_empty() {
+          KIND="$1"
+          NAMES="$(oc -n openshift get -o jsonpath='{.items[*].metadata.name}' "${KIND}")"
```

Comment:
[rehearsal][1]:

```
INFO[2022-06-02T11:10:21Z] environment: line 4: oc: command not found
```

Take a look at [these docs][2] for addressing that.

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29048/rehearse-29048-pull-ci-openshift-cluster-samples-operator-release-4.11-check-aws-no-capabilities/1532307034429460480#1:build-log.txt%3A42
[2]: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli

Reply:
Addressed this issue in the latest commit 44048e9